src/server/command/tp.zig

```diff
@@ -6,93 +6,89 @@ const User = main.server.User;
 pub const description = "Teleport to location.";
 pub const usage = "/tp <x> <y>\n/tp <x> <y> <z>\n/tp <biome>";
 
+const Args = union(enum) {
+	@"/tp <x> <y> <z>": struct {@"<x>": f64, @"<y>": f64, @"<z>": ?f64},
```

Comment:
I do not like how much noise the surrounding `@"<...>"` adds to the parameters, could we maybe apply the angle brackets at the location it is used?

Reply:
It is used in the error messages, so I would suggest to change them to
```diff
- return .initWithFailure(allocator, utils.format(allocator, "Expected a number for {s}, found \"{s}\"", .{name, arg}));
+ return .initWithFailure(allocator, utils.format(allocator, "Expected a number for <{s}>, found \"{s}\"", .{name, arg}));
```
So that we can name the parameters `x` instead of `<x>`.